dmutils/timing.py

```diff
@@ -118,3 +99,58 @@ def logged_duration(
 logged_duration.default_condition = _logged_duration_default_condition
 # exposing this allows a caller to specify default_logger.getChild(...) as their logger
 logged_duration.default_logger = _logged_duration_default_logger
+
+
+def exceeds_slow_external_call_threshold(log_context):
+    """A public condition that will return True if the duration is above the threshold we have defined as acceptable for
+    calls to external services (e.g. Notify, Mailchimp, S3, etc)."""
+    return log_context['duration_real'] > SLOW_EXTERNAL_CALL_THRESHOLD
+
+
+def request_is_sampled(log_context):
+    """A public condition that returns True if the request has the X-B3-Sampled flag set in its headers. While this is
+    the default condition for logged_duration, exposing it publically allows it to be easily combined with other
+    conditions."""
+    return has_request_context() and getattr(request, "is_sampled", False)
```

Comment:
Looks an awful lot like the default condition. Should probably at least make them aliases of each other.

Reply:
I considered that, except the default condition is 'private' and could be changed, and that ideally shouldn't change this 'public' interface.

Maybe I'm being overly pedantic.

Reply:
Fair enough